community/sample-apps/nuxtjs-postgres-graphql/apollo/clientConfig.js

```diff
@@ -1,11 +1,16 @@
 import { InMemoryCache } from "apollo-cache-inmemory";
 export default function(context){
   return {
-  		httpLinkOptions: {
-    		uri: 'http://localhost:8080/v1alpha1/graphql',
-    		credentials: 'same-origin'
-  		},
-  		cache: new InMemoryCache(),
-	    wsEndpoint: 'ws://localhost:8080/v1alpha1/graphql',
-  	}
-}
\ No newline at end of file
+    httpLinkOptions: {
+      // uri: 'https://my-app.herokuapp.com/v1alpha1/graphql',
+      uri: 'http://localhost:8080/v1alpha1/graphql',
+      // headers: {
+      //   'x-hasura-access-key': '<YOUR_SECRET_KEY>'
```

Comment:
Access key should not be present on the client side. May be you can change this to `Authorization`: `Bearer <token>` to reflect that headers can be passed on here for auth.